pkgs/top-level/all-packages.nix

```diff
@@ -26985,9 +26985,7 @@ with pkgs;
     i3GapsSupport = false;
   };
 
-  yambar = callPackage ../applications/misc/yambar { };
-
-  yambar-wayland = callPackage ../applications/misc/yambar { waylandSupport = true; };
+  yambar = callPackage ../applications/misc/yambar { x11Support = true; waylandSupport = true; };
```

Comment:
```suggestion
  yambar = callPackage ../applications/misc/yambar {  };
```
not needed since the defaults are set to true in default.nix
```
, waylandSupport ? true
# Xorg backend
, x11Support ? true
```